README.md

```diff
@@ -4,14 +4,62 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
 `gem 'webmock-fixtures`
 
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "webmock/fixtures"
+
+# Register a fixture as `:get_example`
+WebMock::Fixtures::Manager.register_fixture(:get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+RSpec.describe MyApp do
+  describe "::visit_example" do
+    # Create a manager with the `:get_example` fixture started
+    let!(:manager) { WebMock::Fixtures::Manager.run([:get_example]) }
```

Comment:
While a single class approach works, it leads to everything sharing the same key namespace for fixtures and can lead to bloated files. Are we sure we want to do that?

For reference, in both `httpretty-fixtures` and `fixed-server`/`spy-server`, I opted to using a class per host